docs/concepts/accounts/permission-delegation.md

```diff
@@ -7,102 +7,66 @@ labels:
+Permission delegation is the function of granting various permissions to another account to send permissions on behalf of your account. This can be used instead of or alongside techniques such as [multi-signing](./multi-signing.md) to enable flexible security paradigms such as role-based access control.
 
-## Assigning Permissions
+_(Requires the [PermissionDelegation amendment][] {% not-enabled /%}.)_
 
-You can assign permissions to another account by submitting a `DelegateSet` transaction.
+## Background
 
-```json
-tx_json = {
-  "TransactionType": "DelegateSet",
-  "Account": "rDelegatingAccount",
-  "Authorize": "rDelegatedAccount",
-  "Permissions": [
-    {
-      "Permission": {
-        "PermissionValue": "Payment"
-      } 
-    } 
-  ] 
-} 
-```
+Managing your [cryptographic keys](./cryptographic-keys.md) is one of the more challenging parts of using a blockchain. As part of a defense-in-depth strategy, a secure configuration should limit the damage that can occur if a secret key is compromised. One way to do this is to rotate keys regularly and to keep master keys off of computers that are always connected to the internet and serving user traffic. However, many use cases involve frequently and automatically signing transactions, which typically necessitates having secret keys on an internet-connected server.
 
-| Field | Description |
-|-------|-------------|
-| `Account` | The address of the account that is delegating the permission(s). |
-| `Authorize` | The address of the account that is being granted the permission(s). |
-| `Permissions` | An array of permission objects, specifying the permissions to delegate. Each permission is defined within a `Permission` object, using the `PermissionValue` field. See [XLS-74d, Account Permissions] for a complete list of valid `PermissionValues`. |  
+Permission Delegation can help optimize for these sorts of challenges by granting very limited permissions to separate accounts that can have their keys available online, while keeping the master keys for the main account offline and only using them manually in special cases when rare, important tasks like issuing tokens are necessary. This is especially helpful when using compliance features like [Authorized Trust Lines](../tokens/fungible-tokens/authorized-trust-lines.md) that require a stablecoin issuer to individually approve each user after meeting regulatory requirements like Know Your Customer rules. With a proper configuration, the keys that are used to approve users cannot be used to perform other actions like issuing new tokens or rotating the issuer's key pairs.
 
-## Revoking Permissions
 
-Permissions can be revoked using the `DelegateSet` transaction. There are two ways to revoke permissions:
+## How It Works
 
-### Revoke All Permissions
+The account on whose behalf transactions are being sent is called the _delegating account_. The account sending the transactions is called the _delegate_.
```

Comment:
Is delegating account the official term? Delegator feels more concise.

Reply:
We didn't have an official term. I am fine changing it to delegator.